src/components/saved/SavedFlightPlans.js

```diff
@@ -100,3 +107,26 @@ const SavedItem = styled.li`
     text-align: center;
   }
 `;
+
+const Title = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  h3 {
+    margin-bottom: 5px;
+  }
+  p {
+    margin-bottom: 2px;
+  }
+`;
+
+const Block = styled.div`
+  display: flex;
+  flex-direction: column;
+  span {
+    margin-top: 0px;
```

Comment:
You can get rid of the `px` on 0 measurements. :)

Reply:
Thanks, forget to remove it after testing!